src/app/app.component.spec.ts

```
@@ -14,6 +14,7 @@ describe('AppComponent', () => {
   const MockUserService = {
     getUserName$: jest.fn().mockReturnValue(new Observable()),
   };
+  (window as any).gtag = () => {};
```

Can we suppress this error?

---

showing a lint error `Unexpected any. Specify a different type`

---

resolved, thanks